src/config/localfile-config.c

```diff
@@ -136,6 +138,13 @@ int Read_Localfile(XML_NODE node, void *d1, __attribute__((unused)) void *d2)
             logf[pl].out_format[n]->target = target ? strdup(target) : NULL;
             os_strdup(node[i]->content, logf[pl].out_format[n]->format);
             logf[pl].out_format[n + 1] = NULL;
+        } else if (strcmp(node[i]->element, xml_localfile_reconnect_time) == 0) {
+            int time = atoi(node[i]->content);
```

Comment:
Let's use the standard format in Wazuh for timing settings, allowing a suffix such as `s`, `m`, `h`...